src/SerializerFactory.php

```diff
@@ -94,15 +111,25 @@ private function shouldSerializeReferenceSnaksWithHash() {
 	}
 
 	/**
-	 * Returns a Serializer that can serialize Item and Property objects.
-	 *
-	 * @return Serializer
+	 * @throws IllegalValueException
+	 * @return Serializer A dispatching serializer that can serialize whatever the entity
```

Comment:
(nitpick) Also possibly @return could come before @throws, I think this is the order we often have those.